src/proto/factories/optimizer_factory.cpp

```diff
@@ -43,6 +43,12 @@ namespace lbann {
 namespace proto {
 namespace {
 
+std::unique_ptr<optimizer>
+build_no_optimizer_from_pbuf(
+  google::protobuf::Message const& msg, lbann_comm* comm) {
+  return std::unique_ptr<optimizer>();
```

Comment:
This can just return `nullptr;` All that extra typing... what would @ndryden think?? 😛

Reply:
My poor wrists... are you trying to disable me? 😛